lib/util.js

```diff
@@ -74,24 +81,95 @@ module.exports = (function () {
         return numDevices;
     }
 
+    function getSimId() {
+        var findSimCommand = "cordova run --list --emulator | grep ^iPhone | tail -n1";
```

Comment:
If it's not too hard to implement, i think it would be good to also handle iPad simulators.